### ISSUE TITLE
Build solution gives an error

### ISSUE THREAD

**Issue by Zeeebas** [BenFradet/RiotSharp#686]
I tried to build the repo but it gives these errors. I am fairly sure I am doing something wrong.
Could someone help me out? 

![image](https://user-images.githubusercontent.com/36153482/121904385-6d894d00-cd29-11eb-8fbf-0866626f3039.png)

**Comment by xXLAOKOONXx** [comment]
In the readme you can find an explaination on how to build your own nuget package. After creation you can use the nuget in your other projects then. 
The issues you have I have never seen before. But it looks like following the nuget guide should help. If not maybe try to delete the project and clone it again.

**Comment by Zeeebas** [comment]
Running dotnet pack /p:Configuration=Release in the Developer command prompt gives for even more errors. 

![image](https://user-images.githubusercontent.com/36153482/122035139-77b35600-cdd2-11eb-8ec9-690d0ff3d78b.png)

I tried running these on my PC, (other visual studios 2019) and it works fine. So I guess it's a local thing...


**Edit:**
I made it work and installed the newest .nupkg file on my PC.
I created a local nuget feed on my laptop and transferred the .nugkg file and installed it on my laptop.

This was my first time doing this so it was a bit of a struggle...
